maxim/logger/groq/utils.py

```
@@ -0,0 +1,222 @@
+            if key in kwargs and kwargs[key] is not None and key not in skip_keys:
+                model_params[key] = kwargs[key]
+
+        # Add any other parameters that aren't in skip_keys
+        for key, value in kwargs.items():
+            if key not in param_keys and key not in skip_keys and value is not None:
+                model_params[key] = value
+
+        return model_params
+
+    @staticmethod
+    def parse_chunks_to_response(content: str, usage_data: Optional[CompletionUsage]) -> Any:
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `parse_chunks_to_response`

(ANN401)